src/main/java/org/broadinstitute/barclay/argparser/CommandLineProgramProperties.java

```diff
@@ -25,7 +25,22 @@
      * @return an example command line for this program
      */
     String usageExample() default "The author of this program hasn't included any example usage, please complain to them.";
+
+    /**
+     * Assign this command line program to an (application-defined) program group for use in managing help output
```

Comment:
Similar documentation should be added to the `CommandLineProgramGroup` interface.

Reply:
Done.